pkg/clients/clients.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 var instances struct {
-	Etcd       *etcd.Client
+	Etcd       etcd.Client
```

Comment:
Why is this not a pointer? I moved all the clients to be pointers for reference and consistency. If it's a light object, I don't care, IMO the pointer was cheap and we're referring to the same object anyways with a singleton.